exonum-java-binding/core/rust/integration_tests/tests/transaction_proxy.rs

```diff
@@ -227,19 +227,22 @@ fn passing_transaction_context() {
     let (tx_hash, author_pk) = {
         let mut fork = db.fork();
         let (valid_tx, raw) = create_mock_transaction_proxy(EXECUTOR.clone());
-        // get transaction hash and author public key from mock transaction
         let (tx_hash, author_pk) = {
             let context = create_transaction_context(&mut fork, raw);
             let (tx_hash, author_pk) = (context.tx_hash(), context.author());
 
-            // execute transaction
+            // Transaction will write passed tx_hash and author_PK to
+            // the database (into two `EntryIndex`es)
             valid_tx.execute(context).unwrap();
 
             (tx_hash, author_pk)
         };
         db.merge(fork.into_patch()).unwrap();
         (tx_hash, author_pk)
     };
+
+    // Checking that tx_hash and author_pk passed to the transaction context are
+    // the same as the ones received by transaction during its execution.
```

Comment:
Nit: Sorry, cannot refer directly - at line 262 we have this:
```
let (pk, sk) = java_bindings::exonum::crypto::gen_keypair();
let signed_transaction = java_bindings::exonum::messages::Message::sign_transaction(
```
We already have `java_bindings::exonum::crypto` and `java_bindings::exonum::messages` in the use section, perhaps we could also extend then with these entities.